package.json

```diff
@@ -43,6 +43,7 @@
         "jquery": "^3.7.1",
         "jquery-migrate": "^3.5.2",
         "jquery-prettytextdiff": "^1.0.4",
+        "jquery-serializejson": "^3.2.1",
```

Comment:
I am not in favor of adding a new jQuery plugin for this.
I think it could be done easilly with somthing like this:
```js
const data = new FormData(document.getElementById('main-form'));
let object = {};
for ([key, value] of data.entries()) {
    object = _.set(object, key, value);
}
console.log(object);
```